meshtastic/mesh_interface.py

```diff
@@ -9,17 +9,18 @@
 import threading
 import time
 from datetime import datetime
-from typing import AnyStr
 
 import google.protobuf.json_format
-import timeago
+import timeago # type: ignore[import-untyped]
 from google.protobuf.json_format import MessageToJson
-from pubsub import pub
+from pubsub import pub # type: ignore[import-untyped]
```

Comment:
These sorts of `ignore[import-untyped]` silence mypy a bit when using libraries that don't have type hints (or at least, aren't marked as such) nor type stubs.